packages/run-protocol/src/vaultFactory/liquidateMinimum.js

```diff
@@ -24,24 +24,26 @@ const trace = makeTracer('LM');
 const start = async zcf => {
   const { amm } = zcf.getTerms();
 
-  /**
-   * @param {Amount<'nat'>} runDebt
-   */
-  const makeDebtorHook = runDebt => {
-    const runBrand = runDebt.brand;
-    return async debtorSeat => {
+  const makeDebtorHook = _runDebt => {
+    /**
+     * @param {ZCFSeat} debtorSeat
+     * @param {object} root0
+     * @param {Amount<'nat'>} root0.debt
```

Comment:
I think this is the prevalent style and less verbose:
```suggestion
     * @param {{ debt: Amount<'nat'> }} options
```